proxysql-admin

```diff
@@ -1507,7 +1507,20 @@ function enable_proxysql() {
                            "\n-- Please check the ProxySQL connection parameters and status."
     fi
   fi
+  
```

Comment:
This should probably be a function (taking the server to use as a parameter).

Reply:
Added update_mysql_version function